services/s3/src/main/resources/codegen-resources/customization.config

```diff
@@ -4,15 +4,9 @@
         "Object" : "S3Object"
     },
     "serviceSpecificClientConfigClass": "S3Configuration",
-    "shapeModifiers" : {
-        "GetBucketPolicyOutput" : {
-            "customUnmarshallerFqcn": "software.amazon.awssdk.services.s3.transform.GetBucketPolicyResponseUnmarshaller"
```

Comment:
Don't we need this?

Reply:
I added an interceptor to deal with this

Reply:
Handwritten GetBucketPolicyResponseUnmarshaller class is not deleted, I got confused by that.

Reply:
Oops, I'll delete that. Good catch.